src/services/db/GitHubService.ts

```diff
@@ -17,6 +17,8 @@ import logger from "@root/logger/logger"
 import { GitCommitResult } from "@root/types/gitfilesystem"
 import { GitHubRepoInfo, RawGitTreeEntry, RepoState } from "@root/types/github"
 
+import { E2E_TEST_GH_TOKEN } from "../middlewareServices/AuthenticationMiddlewareService"
```

Comment:
do we actually have this token at run time or is it only during e2e on local

Reply:
Run time - we currently also use it for pr approval

Reply:
Not a blocker but The name of the token is very misleading